fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -610,7 +613,16 @@ public String logKey() {
         DELETES("deletes", false),
         /** Total number of mutation operations. */
         MUTATIONS("mutations", false),
-        ;
+        /** JNI Calls in FDB.*/
+        FDB_JNI_CALLS("FDB jni calls",false),
+        /**Bytes read from FDB.*/
+        FDB_BYTES_FETCHED("FDB bytes fetched",false),
+        /** Number of network fetches performed by FDB.*/
+        FDB_RANGE_QUERY_FETCHES("FDB Range query fetches",false),
+        /** Number of records fetched during a range scan.*/
+        FDB_RANGE_QUERY_RECORDS_FETCHED("FDB range records ",false ),
```

Comment:
`QUERY_RECORDS` is a bit misleading.  Record Layer has a concept of records, but I don't think this aligns with this concept.  Maybe we should call it `FDB_RANGE_KEYVALUES_FETCHED` or something like that?